indigo-plugin/indigo-plugin/src/indigoplugin/Utils.scala

```diff
@@ -6,7 +6,8 @@ final case class TemplateOptions(
     title: String,
     showCursor: Boolean,
     scriptPathBase: Path,
-    gameAssetsDirectoryPath: Path
+    gameAssetsDirectoryPath: Path,
+    backgroundColor: String = "white"
```

Comment:
Do we need the default? Can it be removed? I suspect it's unnecessary? SBT will default to `white` and Mill forces you to provide a value.